src/VariableCollector/DefaultTemplateVariables.php

```diff
@@ -21,6 +21,7 @@ public function collect(): array
         $variables[] = new Variable('baseUrl', new StringType());
         $variables[] = new Variable('basePath', new StringType());
         $variables[] = new Variable('ʟ_fi', new ObjectType('Latte\Runtime\FilterInfo'));
+        $variables[] = new Variable('ʟ_tag', new ArrayType(new MixedType(), new MixedType()));
```

Comment:
don't we know the exact or more strict type of this variable?

Reply:
I think it is string[] but not quite sure without closet examination of latte internals

Reply:
It seem sot be string[] I will narrow it down.